### DISCUSSION TITLE
[@types/braintree] Mispelled type additionalProccessorResponse

### DISCUSSION THREAD

**Discussion by bryevo** [DefinitelyTyped/DefinitelyTyped#65478] [Issues with a @types package]
In the Transaction object, the `additionalProccessorResponse` attribute is mispelled.  The attribute name should be `additionalProcessorResponse`.

A valid transaction payload returns `additionalProcessorResponse` as the attribute key name.

[a comment by an automated account is not shown]

**Comment by smrubin** [comment]
@bryevo Would you be willing to open a PR for this?

**Comment by MysteryBlokHed** [comment]
I just opened #65497
